script/test

```diff
@@ -95,21 +95,53 @@ py_unittests() {
   cd -
 }
 
-do_go_tests=0
-do_py_unittests=0
-
-for var in "$@"; do
-  if [[ "$var" == "go-tests" ]]; then
-    do_go_tests=1
-  elif [[ "$var" == "py-unittests" ]]; then
-    do_py_unittests=1
+check_py_example() {
+  echo "Checking pyOTNS example: $1 ..."
```

Comment:
nit: openthread scripts are using indention with 4 spaces.

Reply:
yes, I am planing to use the `shfmt` to format all shell scripts, same like `openthread`. But it worth a separate PR.